modules/nw-allocating-load-balancers-to-subnets.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// *networking/allocating-load-balancers.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-allocating-load-balancers-to-subnets_{context}"]
+= Allocating API and Ingress Load Balancers to Specific Subnets
```

Comment:
Should we mention an update case when an existing `IngressController` is updated with new subnets? It's a particular case which needs the user's manual intervention to recreate the OpenShift service.